app/javascript/article-form/components/ArticleCoverImage.jsx

```diff
@@ -89,7 +89,7 @@ export class ArticleCoverImage extends Component {
     e.preventDefault();
     const { onMainImageUrlChange } = this.props;
     onMainImageUrlChange({
-      links: [],
+      links: [null],
     });
   };
 
```

Comment:
I wasn't sure if I should write a Jest test or an E2E test to test this feature... Maybe @forem/frontend could give some guidance?